apps/src/lib/kits/maker/CircuitPlaygroundBoard.js

```diff
@@ -385,9 +385,49 @@ export default class CircuitPlaygroundBoard extends EventEmitter {
       ? SerialPort
       : ChromeSerialPort.SerialPort;
 
-    return new SerialPortType(portName, {
+    const port = new SerialPortType(portName, {
       baudRate: SERIAL_BAUD
     });
+
+    if (isNodeSerialAvailable()) {
+      const queue = [];
+      let sendPending = false;
```

Comment:
Should `sendPending` be a property on `port`?

Reply:
And `queue`?

Reply:
👍

Reply:
Ah sorry, I may have led you astray here. I'm not sure we want to hang these properties on `port` since it's the `SerialPort` instance and there's a tiny chance we could overwrite future public properties on that instance.

Is there a reason to avoid having these as local variables? The should get pulled into the closure so they're unique per invocation of `openSerialPort`.